### PR TITLE
[dashboard] Remove duplicate status header in Prebuild page

### PR DESCRIPTION
**Pull request by jankeromnes**
Implements B from https://github.com/gitpod-io/gitpod/issues/5526

## Description

| BEFORE | AFTER |
| --- | --- |
| Screenshot 2021-09-17 at 09 59 09 | Screenshot 2021-09-17 at 09 56 36 |

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/5526

## How to test

1. Trigger one or more prebuilds for a project
2. Open the prebuild page -- there should be only one status widget shown

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **jankeromnes** after the PR has been reviewed.

Associated issue: *#5526*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by:

Associated issue: *#5526*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jankeromnes** [comment]
| BEFORE | AFTER |
| --- | --- |
| Screenshot 2021-09-17 at 09 59 09 | Screenshot 2021-09-17 at 09 56 36 |

**Review by gtsiolis — APPROVED** [review]
Looks straightforward and seems to resolve https://github.com/gitpod-io/gitpod/issues/5526!

Thanks @jankeromnes! Welcome back! 🇬🇷 

Deleted code is the best code. 📕

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 815dcd84859c9a7d34b5f7aa8c9d589e63a51edc</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5730#pullrequestreview-757466711" title="Approved">gtsiolis</a>*

Associated issue: *#5526*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
